x-pack/plugins/fleet/dev_docs/data_model.md

```diff
@@ -46,25 +51,42 @@ list.
 
 The total schema for actions is represented by the `FleetServerAgentAction` type.
 
+- Cleanup model: Fleet Server considers actions expired after 30 days, and will remove them via an hourly process
+- [Source](https://github.com/elastic/fleet-server/blob/9af3b2176b42a0de34c5583b5430558c03792dd0/internal/pkg/gc/schedules.go#L29-L33)
+
 ### `.fleet-actions-results`
 
+- Cleanup model: N/A
+
 ### `.fleet-servers`
 
+- Cleanup model: N/A
```

Comment:
I don't see any logic in Kibana or Fleet Server for removing documents from the `.fleet-server` index. I'm not sure if we should remove documents when a Fleet Server goes offline, but that's my only thought for a cleanup model here.

Reply:
How is this index actually used? Without that it's hard to answer how this be cleaned up. That said, it's probably low prio since this should be a relatively small index. The case where it may be more problematic is if you're running FS in a container that is getting rescheduled constantly.